### DISCUSSION TITLE
Chatting over documents.

### DISCUSSION THREAD

**Discussion by onebefa** [llm-workflow-engine/llm-workflow-engine#289] [General]
Hello, I'm playing with langchain at the moment and embeddings. Is there any possibility to use the Wrapper (browser-mode) with the embeddings. My embeddings are a series of HTML pages stored locally.

If we can't use the embeddings, is there any other solution with the wrapper?

I saw that there is this function: data_query: Send natural language commands to a loaded file of structured data

Is it possible to use it with multiple HTML pages (in browser mode)?

**Comment by thehunmonkgroup** [comment]
> Is there any possibility to use the Wrapper (browser-mode) with the embeddings. My embeddings are a series of HTML pages stored locally.
> 
> If we can't use the embeddings, is there any other solution with the wrapper?

Not currently. It might be fairly easy to write a command plugin for that, leveraging Langchain. Would need more investigation.

> I saw that there is this function: data_query: Send natural language commands to a loaded file of structured data

This is for files like JSON, CSV, etc.

> Is it possible to use it with multiple HTML pages (in browser mode)?

No idea what this means.

**Reply by onebefa** [reply]
Hi !

Thanks for your reply, it would be very nice if the wraper can be implemented with Langchain !

Let me know ;-)

**Comment by thehunmonkgroup** [comment]
This is an open source project, I encourage you to submit PRs for the features you want to see, and/or sponsor the work.
